content/en/contribute/medic/onboarding/product-team.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Product Team"
+linkTitle: "Product Team"
+weight: 3
+description: >
+  Introduction to Product Team 
+---
+
+The goal of this page is to convey a general “lay of the land” so someone starting can see a lot of what’s out there related to Product Team without having to be surprised each day as new things pop up.
+
+## About Product Team
+The Product Team as a whole is focused on digital health tools helping to achieve Universal Health Coverage, and care reaching everyone when and where they need it. 
+We achieve this by developing and maintaining the CHT in different ways:
+- Building tools
+- Building community 
+- User research and testing
+- Collaborations with partners, usually non-profit organization and government's ministries or institutions. 
+- Internal cross-functional collaborations with Programs and Research teams. 
+
+## Composition
+The Product team is made up of a community team and 4 focused groups, with each group focusing on outcomes to serve different people.
+
+### Focus Groups
+The Product Team operates in a way that is most commonly referred to as Continuous Discovery. That process helps us to focus on the impact of our work and keep teammates well-aligned with the people who use the CHT. Read more about the process [here]({{< ref "contribute/medic/product-development-process" >}}). 
+
+Each focus group serve to different user types to understand opportunities for improvement, find solutions that will have an impact, and then build, deploy, and measure them. Find out more about the different focus groups [here]({{< ref "contribute/medic/product-development-process/focused-groups" >}}).
+
+### Community Team
```

Comment:
@esthermmoturi We will have new teammates joining Medic. The intention of this page is to introduce topics related to the Product Team as a compliment to the onboarding process. This is just a start, and we will be adding more things. 
Can you please help me fill up this piece? What can we say about the Community Team to new teammates? Some ideas: 
- What is Community team?
- Why do we have a community team?
- What are Round ups? - link to previous round up calls? 
- etc